src/components/HelpDrawer/HelpContent/SpectralProfilerSettingsMomentsHelpContent.tsx

```diff
@@ -40,16 +42,13 @@ export const SPECTRAL_PROFILER_SETTINGS_MOMENTS_HELP_CONTENT = (
             </li>
         </ul>
         <p>
-            When all the parameters are defined, by clicking the <code>Generate</code> button moment calculations will begin. Depending on the file size, moment calculations may take a while. If that happens, you may consider to cancel the
+            When all the parameters are defined, by clicking the <code>Generate</code> button moment calculations will begin. Depending on the file size, moment calculations may take a while. If that happens, you may consider canceling the
```

Comment:
"click the <code>Generate</code> button to start the moment calculation"

"may wish to cancel"